src/VbsEnclaveSDK/src/veil_enclave_lib/userboundkey.vtl1.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <cstdint>
+#include <wil/resource.h>
+#include "future.vtl1.h"
+#include <veinterop_kcm.h>
+#include <stdexcept>
+#include <VbsEnclave\Enclave\Implementation\Types.h>
+
+namespace veil::vtl1::userboundkey
+{
+    struct keyCredentialCacheConfig
+    {
+        std::uint32_t cacheOption {};
+        std::uint32_t cacheTimeoutInSeconds {};
+        std::uint32_t cacheUsageCount {};
+    };
+
+    wil::secure_vector<uint8_t> create_user_bound_key(
+        const std::wstring& keyName,
+        veil::vtl1::userboundkey::keyCredentialCacheConfig& cacheConfig,
```

Comment:
>cacheConfig [](http://example.com/codeflow?start=60&length=11)

const on both cacheConfigs

Reply:
because you didn't make **.value** const in your param struct.

propCacheConfig**.value** = &cacheConfig.

We can't fix it now, but at least you can put a const in the signature to create_user_bound_key/load_user_bound_key so we don't force our const-correct mistake on our poor callers.